syntax = "proto3";
package stream;




import "metadata_center_sdk/model/metadata_center/stream_metric_schema.proto";

/*
ListMetricStates返回
*/
message ListMetricStatesResponse {
    /*
    指标项列表
    */
    repeated metadata_center.StreamMetricSchema states = 1 ;
}

/*
ListMetricStatesApi返回
*/
message ListMetricStatesResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    stream.ListMetricStatesResponse data = 4 ;
}

